_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,284 @@
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="new_job_01">
+                <input type="checkbox" class="input input-checkbox" name="edx_goals_01" id="new_job_01" value="I need a new jobby job!" checked="checked">
+                <span class="input-label">I need a new jobby job!</span>
+            </label>
+        </div>
+    </fieldset>
+    <fieldset class="fieldset form-group">
+        <legend class="group-legend form-label">Enhanced checkboxes</legend>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="outsmart_prof_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="outsmart_prof_02" value="Outsmart those professors" checked="checked">
+                <span class="input-label">Outsmart those professors</span>
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="try_and_see_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="try_and_see_02" value="Try it out and see what I can learn">
+                <span class="input-label">Try it out and see what I can learn</span>
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="new_job_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="new_job_02" value="I need a new jobby job!" checked="checked">
+                <span class="input-label">I need a new jobby job!</span>
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Textareas</h4>
+<div class="example-set">
+    <fieldset class="fieldset">
+        <legend class="group-legend is-hidden">Storytime!</legend>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_01">Short textarea</label>
+            <textarea class="textarea input-textarea textarea-short" name="textarea_01" id="textarea_01"></textarea>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_02">Base textarea</label>
+            <textarea class="textarea input-textarea textarea-medium" name="textarea_02" id="textarea_02"></textarea>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_03">Tall textarea</label>
+            <textarea class="textarea input-textarea textarea-tall" name="textarea_03" id="textarea_03"></textarea>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_04">Block textarea</label>
+            <textarea class="textarea input-textarea textarea-medium textarea-block" name="textarea_04" id="textarea_04"></textarea>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Sliders</h4>
```

Comment:
Add ``.example-set-hd`` class in here for consistency and extra PL Doc site formatting.